py3status/modules/file_status.py

```diff
@@ -107,7 +104,7 @@ def post_config_hook(self):
         # convert str to list + expand path
         if not isinstance(self.paths, list):
             self.paths = [self.paths]
-        self.paths = list(map(expanduser, self.paths))
+        self.paths = [path.expanduser() for path in self.paths]
```

Comment:
I may have read through this too quick but how are the paths in self.path magically converted to Path so you can use expanduser() like this?

Reply:
That's what reviews are for :+1: Thank you for checking it.

Fixed to

```
        self.paths = [Path(path).expanduser() for path in self.paths]
```

Is there actually no test coverage run in tox?

Reply:
I think this mistake is present on some other files, I'll get through them again then

> Is there actually no test coverage run in tox?

not those ones no; we've always been willing to but..